include/materials/norsand.tcc

```diff
@@ -540,7 +568,7 @@ Eigen::Matrix<double, 6, 1> mpm::NorSand<Tdim>::compute_stress(
   (*state_vars).at("plastic_strain5") += dpstrain(5);
 
   // Update equivalent plastic deviatoric strain
-  (*state_vars).at("epds") =
+  (*state_vars).at("pdstrain") =
```

Comment:
I'm not sure about the coefficient in the equation of pdstrain, and I think the epds and pdstrain have different coefficients

Reply:
Shouldn't it be the same?

Reply:
Yes, thus we use the word `pdstrain` in this case